youtube_dl/extractor/ard.py

```diff
@@ -127,8 +127,8 @@ class ARDMediathekIE(ARDMediathekBaseIE):
         'info_dict': {
             'id': '44726822',
             'ext': 'mp4',
-            'title': 'Was ist die Kunst der Zukunft, liebe Anna McCarthy?',
-            'description': 'md5:4ada28b3e3b5df01647310e41f3a62f5',
+            'title': 'SÜDLICHT: Was ist die Kunst der Zukunft, liebe Anna McCarthy? | ARD Mediathek',
```

Comment:
Basically, there should not be suffixes like ` | ARD Mediathek` so that title extraction should be fixed.

Reply:
Thanks! I actually went with trying the OpenGraph attributes instead of changing the test.